src/test/regress/expected/multi_explain.out

```diff
@@ -497,19 +497,16 @@ EXPLAIN (COSTS FALSE)
 	WHERE l_orderkey = o_orderkey
 	AND o_custkey = c_custkey
 	AND l_suppkey = s_suppkey;
-Distributed Query into pg_merge_job_570021
+Distributed Query into pg_merge_job_570020
   Executor: Task-Tracker
   Task Count: 1
   Tasks Shown: None, not supported for re-partition queries
   ->  MapMergeJob
-        Map Task Count: 1
+        Map Task Count: 8
         Merge Task Count: 1
-        ->  MapMergeJob
-              Map Task Count: 8
-              Merge Task Count: 1
```

Comment:
We're losing test coverage on explaining nested jobs here, could we get that back somehow?